app/src/pages/inside/logsPage/historyLine/historyLine.jsx

```diff
@@ -101,7 +108,8 @@ export class HistoryLine extends Component {
       historyItems.length < HISTORY_DEPTH_LIMIT &&
       loadedItems >= 0 &&
       loadedItems % NUMBER_OF_ITEMS_TO_LOAD === 0;
-    this.setState({ isLoading: false, shouldShowLoadMore });
+    this.setState({ isLoading: false });
+    this.props.setShouldShowLoadMoreAction(shouldShowLoadMore);
```

Comment:
Can this calculation be done in sagas? I think it make sense to dealing with it in the `fetchHistoryItems` saga, when we just receive necessary data from BE.
This fix also will help us to avoid redundant changes in other components (LogToolbar, LogsPage).